Core/EventHandlers.lua

```diff
@@ -2082,5 +2088,22 @@ function Rarity:OnDisgustingVatFished()
 	addAttemptForItem("Emmah", "pets")
 end
 
+local dreamseedMounts = {
+	[L["Reins of the Winter Night Dreamsaber"]] = true,
```

Comment:
Item names can't be localized here. The database uses English names internally, so on non-English client locales this will not find the item and thus fail to add any attempts.

Reply:
isnt another item using the same format?

Reply:
Where? I'm only seeing English names used with `addAttemptForItem` - which is what I'd expect.

Note that the object names do need to be localized, but internally the English name is used for the item database.